safenode/src/bin/safe/cli/mod.rs

```diff
@@ -10,12 +10,17 @@ mod register;
 mod wallet;
 
 use clap::{Parser, Subcommand};
+use libp2p::Multiaddr;
 
 pub(super) use self::{files::files_cmds, register::register_cmds, wallet::wallet_cmds};
 
 #[derive(Parser)]
 #[command(author, version, about, long_about = None)]
 pub(super) struct Opt {
+    /// Nodes we dial at start to help us get connected to the network. Can be specified multiple times.
```

Comment:
I would suggest a couple of little changes for this.

If we're going to allow the argument to be used multiple times, perhaps it should be renamed `peer` singular. Otherwise, if it's `peers`, we should probably tell the user how each of the multiaddr values are separated, i.e., by spaces, colons or whatever.

For the UI text, I would suggest:
```
Provide a peer to connect to a public network. The argument can be used multiple times and the peer should be specified in the 'MultiAddr' format.

If not provided, a connection will be attempted to a local network.
```

I realise it's just been copied from the node, but I don't think the node is necessarily right either.

Reply:
@jacderida, I tried understand what change you requested exactly, so that I could push it. But didn't quite.
So I will merge this without the change, because I think we might need this for a bug we're seeing. But if you raise PR for it, we'll get it right in!